plugins/active-directory/named_actions.yaml

```
@@ -248,15 +248,15 @@ named_actions:
       contentType: application/json
       url: '{{"https://graph.microsoft.com/v1.0/groups/" + params.group_id + "/members/$ref"}}'
 
-  - name: RemoveGroupMemeber
+  - name: RemoveGroupMember
```

```suggestion
  - name: RemoveGroupMemeber
```
sadly, we're gonna have to revert this at this time, as every playbook containing the action "active-directory.RemoveGroup**Memeber**" will break (remember, actions are referenced by full name in the automation yaml, so every change requires a migration)